lib/db.js

```diff
@@ -1,12 +1,13 @@
 const debug = require('debug')('cosa:db');
 const defaults = require('defaults');
-const EJSON = require('mongodb-extended-json');
+const { ObjectId } = require('bson');
```

Comment:
is there a difference between requiring ObjectID vs ObjectId? is one the correct one vs the other?

Reply:
Nope exactly the same. ObjectId gets exported as both ObjectId and ObjectID. 
https://github.com/mongodb/js-bson/blob/master/src/bson.ts#L84

Reply:
looking it looks like they would prefer you use `ObjectId` - is that correct? if so we should just go ahead and make everything in here consistent with ObjectId

Reply:
```
// In 4.0.0 and 4.0.1, this property name was changed to ObjectId to match the class name.
  // This caused interoperability problems with previous versions of the library, so in
  // later builds we changed it back to ObjectID (capital D) to match legacy implementations.
 ```
 
 kind of seems like it doesnt matter to them. but I guess we can to keep it consistent.